### PR TITLE
fix #73297 cuntwars.com

### LINKED ISSUE #73297: cuntwars.com
### Issue URL (Annoyance)
[NSFW] [https://cuntwars.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fcuntwars.com%2F)
### Comment
Username: @wernser412
> 

### [NSFW] Screenshots
<details>
  <summary>Screenshot 1</summary>

  [Screenshot 1](https://reports-img.adguard.com/zoLq4tR.png)
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.31
Browser: | Chrome
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard Russian, <br/>AdGuard Spanish/Portuguese

### PR DESCRIPTION
**Pull request by stanislav-atr**
#73297

Should we also block `||offers.hooligapps.com/xoffers.js`?

### CODE REVIEW

File: EnglishFilter/sections/specific.txt [hunk 1]
```diff
@@ -269,6 +269,8 @@ nba.com##div[class^="DisplayAd_"]
 modsfire.com##center > a[target="_blank"] > img
 msfsaddon.com###after-ad
 avforums.com##.m2n-ads-slot
+blitzadultparty.ru##a[href*="banner_footer"][target="_blank"]
+blitzadultparty.ru##.bnr
```
Comment (zloyden) [inline EnglishFilter/sections/specific.txt:273]: Why not `blitzadultparty.ru##.cross-promo-bnr`?

**Comment by zloyden** [comment]
> Should we also block `||offers.hooligapps.com/xoffers.js`?

I don't think so. Why do you want to block it?

File: EnglishFilter/sections/specific.txt [hunk 2]
```diff
@@ -269,6 +269,8 @@ nba.com##div[class^="DisplayAd_"]
 modsfire.com##center > a[target="_blank"] > img
 msfsaddon.com###after-ad
 avforums.com##.m2n-ads-slot
+blitzadultparty.ru##a[href*="banner_footer"][target="_blank"]
```
Comment (zloyden) [inline EnglishFilter/sections/specific.txt:272]: Do you still need it? Last one is enough.

**Comment by stanislav-atr** [comment]
> > Should we also block `||offers.hooligapps.com/xoffers.js`?
> 
> I don't think so. Why do you want to block it?

@zloyden, it kinda uses info about ads, not sure to be honest.

**Review by zloyden — APPROVED** [review]

**Comment by Yuki2718** [comment]
> 
>  it kinda uses info about ads, not sure to be honest.

Probably better not to block. These online games have promotion system which user get in-game currency if he click occasional in-game ads. However, I think `||analytic-client.playful-fairies.com^` can be added to Spyware, apparently doesn't break the game.

**Comment by Yuki2718** [comment]
https://github.com/AdguardTeam/AdguardFilters/commit/3a825f1aa877977e6a26e45bb9b2d627e4a7de7c

**Pull request merged**